doc/input-output-reference/src/overview/group-internal-gains-people-lights-other.tex

```diff
@@ -101,29 +90,15 @@ \subsubsection{Inputs}\label{inputs-025}
 
 This field is a key/choice field that tells which of the next three fields are filled and is descriptive of the method for calculating the nominal number of occupants (people) in the Zone. The key/choices are:
 
-\begin{itemize}
-\tightlist
-\item
-  People
-\end{itemize}
-
-With this choice, the method used will be a straight insertion of the number of occupants (people).~ (The Number of People field should be filled.)
-
-\begin{itemize}
-\tightlist
-\item
-  People/Area
-\end{itemize}
+\begin{description}
+	
+\item[People] With this choice, the method used will be a straight insertion of the number of occupants (people).~ (The Number of People field should be filled.)
 
-With this choice, the method used will be a factor per floor area of the zone. (The People per Zone Floor Area field should be filled).
+\item[People/Area] With this choice, the method used will be a factor per floor area of the zone. (The People per Zone Floor Area field should be filled).
 
```

Comment:
"floor area of the zone. (The People per Zone Floor Area" Since there is more work here, I would suggest adding to the end of the first sentance "or Space" in the future here and other places like this. Also it's just "People per Floor Area" now, another clean up that could happen in a future commit.